src/pages/hyeriShin/Login/LoginShin.js

```diff
@@ -1,8 +1,95 @@
 import React, { Component } from 'react';
-//import { withRouter } from 'react-router';
+import './LoginShin.scss';
+
 class LoginShin extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      idInput: '',
+      pwInput: '',
+      name: 'jetom',
+      phone_number: '010-0408-1111',
+      gender: 'Male',
+    };
+  }
+
+  goToMainShin = e => {
+    e.preventDefault();
+
+    fetch('http://10.58.3.243:8000/users/login', {
+      method: 'POST',
+      body: JSON.stringify({
+        email: this.state.idInput,
+        password: this.state.pwInput,
+        name: this.state.name,
+        phone_number: this.state.phone_number,
+        gender: this.state.gender,
+      }),
+    })
+      .then(response => response.json())
+      .then(result => console.log('결과: ', result));
+    // this.props.history.push('/MainShin');
+  };
+
+  handleInput = e => {
+    const { value, name } = e.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
   render() {
-    return null;
+    const isLoginBtn =
+      this.state.idInput.includes('@') && this.state.pwInput.length >= 5;
+    const { handleInput, goToMainShin } = this;
+
+    return (
+      <div className="loginBody">
+        <div className="cotainer">
+          <header>
+            <h1 className="logo">
+              Tomstagram <i className="fas fa-dog"></i>
+            </h1>
+          </header>
+
+          <section>
+            <article>
+              <div className="loginBox">
+                <form onSubmit={goToMainShin}>
+                  <input
+                    name="idInput"
+                    onChange={handleInput}
+                    type="text"
+                    placeholder="전화번호, 사용자 이름 또는 이메일"
+                    className="loginInput info"
+                  />
```

Comment:
```suggestion
                  <input
                    name="idInput"
                    onChange={handleInput}
                    placeholder="전화번호, 사용자 이름 또는 이메일"
                    className="loginInput info"
                  />
```
type="text"는 디폴트 값입니다

Reply:
테..ㄱ..스..트.....는.. 디폴트 값... 메모 했습니다 ✍️